back-end/routes/api/users.js

```diff
@@ -29,16 +29,20 @@ router.post("/register", async (req, res) => {
   }
 
   // Try to find an existing user
-  let user = await User.findOne({ email: req.body.email });
+  let user = await User.findOne({ email: req.body.email, username: req.body.email });
 
   // If there is a user, return
   if (user) {
     return res.status(400).json({ email: "Email already exists" });
   }
+  if (user) {
```

Comment:
You could change these `if` statements with the new variables, one for a user with an email that already exists and one for a user with a username that already exists

Reply:
This has been updated 😄